### ISSUE TITLE
Unexpected token

### ISSUE THREAD

**Issue by xg-wang** [gabrielcsapo/docusaurus-plugin-api-extractor#13]
Hi, I'm trying to set up a project with api-extractor and docusaurus, but hitting this error:

```
SyntaxError: /Users/xwang5/Code/beacon-transporter/website/docs/api/beacon-transporter.beaconfunc.md: Unexpected token (14:81)
  12 | <pre><code parentName="pre" {...{"className":"language-typescript"}}>{`export declare type BeaconFunc = (url: string, body: string, headers?: Record<string, string>) => Promise<RetryRejection | RequestSuccess | undefined>;
  13 | `}</code></pre>
> 14 | <b>References:</b> [RetryRejection](/docs/api/beacon-transporter.retryrejection)<!-- -->, [RequestSuccess](/docs/api/beacon-transporter.requestsuccess)
     |                                                                                  ^
  15 |     </MDXLayout>
  16 |   )
  17 | };

```

Here's the source https://github.com/xg-wang/beacon-transporter/blob/3bea3202227aed02457e310c98846ec65530c93c/docs/beacon-transporter.beaconfunc.md

**Comment by chadhietala** [comment]
I believe I fixed this in #14. I'll let you know when this gets released.

**Comment by gabrielcsapo** [comment]
Please reopen if this is not fixed
